src/client/hud.h

```diff
@@ -18,8 +18,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-#ifndef CLIENT_HUD_HEADER
-#define CLIENT_HUD_HEADER
```

Comment:
I'm pretty sure the `#ifndef` was left untouched on purpose, because there are two header files with the same name:

- `src/hud.h`
- `src/client/hud.h`

Would `#pragma once` take this quirk into account?

Reply:
From what I've gathered from [here](https://gcc.gnu.org/onlinedocs/cpp/Pragmas.html), [here](https://en.wikipedia.org/wiki/Pragma_once) and [here](https://docs.microsoft.com/en-us/cpp/preprocessor/once?view=vs-2019), I think this quirk should be handled correctly.

Reply:
That does indeed seem to be the case, thanks for clarifying.